pkg/ui/builder.go

```diff
@@ -0,0 +1,77 @@
+package ui
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/fatih/color"
+)
+
+var red = color.New(color.FgRed)
+
+type ConsoleUiBuilder struct {
```

Comment:
Question: Do we really need a Builder? It seems that we can support everything with the UserInterface interface. I see the possibility to reduce the amount of code here.